src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  margin: auto;
+  flex-wrap: wrap;
+  gap: $base-gap;
+  width: ($square-size * 7) + ($base-gap * 6);
```

Comment:
The width of the calendar is calculated correctly, but it would be better to use a variable for the number of days in a week for better readability and flexibility. For example, you can declare a new variable `$days-in-week: 7;` in your `variables.scss` file and use it here like so: `width: ($square-size * $days-in-week) + ($base-gap * ($days-in-week - 1));`